.eslintignore

```diff
@@ -0,0 +1,14 @@
+.yarn/**/*
+**/node_modules/**/*
+docs/**/*
+packages/*/examples/**/*
+packages/*/test/**/fixtures/**/*
+packages/*/test/projects/**/*
+packages/core/lib/*.umd.js
+packages/lavapack/bundle.js
+packages/lavapack/src/runtime-cjs.js
+packages/lavapack/src/runtime.js
+packages/perf/trials/**/*
+packages/survey/mitm/**/*
+packages/viz/dist/**/*
+packages/viz/src/example-policies/**/*
```

Comment:
I'm assuming this is extracted so that other tools pick it up. Otherwise we'd keep it in one file to minimize the number of top-level files.
Correct me if I'm wrong.

Reply:
Yes.